content/concept-amendments.md

```diff
@@ -235,11 +248,10 @@ Changes the hash tree structure that `rippled` uses to represent a ledger. The n
 
 | Amendment ID                                                     | Status    |
 |:-----------------------------------------------------------------|:----------|
-| DA1BD556B42D85EA9C84066D028D355B52416734D3283F85E216EA5DA6DB7E13 | Released but not enabled <br />Enabled on TestNet |
+| DA1BD556B42D85EA9C84066D028D355B52416734D3283F85E216EA5DA6DB7E13 | Withdrawn for production.<br />Enabled on TestNet |
```

Comment:
Maybe it would be worth adding a section that explains the statuses? We could adapt our internal Confluence page (though we'd still have to add "withdrawn").